### PR TITLE
chmod: add Indonesian translation

### PR DESCRIPTION
**Pull request by reinhart1010**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by EmilySeville7cfg — APPROVED** [review]
Translation is good. ;)

But the [original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/chmod.md) has several issues:

- wrong placeholder: `{{file}}` instead of `{{path/to/file}}`
- wrong placeholder: `{{directory}}` instead of `{{path/to/directory}}`

**Comment by EmilySeville7cfg** [comment]
#8300 fixes original page.

File: pages.id/common/chmod.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# chmod
+
+> Mengubah hak akses pengguna ([u]ser) suatu file atau direktori.
```
Comment (marchersimon) [inline pages.id/common/chmod.md:3]: How did the `[u]ser` get in there? This isn't in the original page.

Comment (navarroaxel) [reply pages.id/common/chmod.md:3]:
```suggestion
> Mengubah hak akses pengguna suatu file atau direktori.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**